lib/api/initiateMultipartUpload.js

```diff
@@ -134,7 +134,16 @@ function initiateMultipartUpload(authInfo, request, log, callback) {
                 }
                 services.metadataStoreMPObject(MPUbucket.getName(),
                     cipherBundle, metadataStoreParams,
-                    log, err => callback(err, xml));
+                    log, err => {
+                        if (err) {
+                            log.trace('error storing multipart object', {
+                                error: err,
+                            });
+                            return callback(err);
+                        }
+                        log.trace('successfully initiated mpu', { uploadId });
```

Comment:
is https://github.com/scality/S3/pull/579/commits/356d8b2dddb854d54d8a3a26e2511a9a854b1b8d better?